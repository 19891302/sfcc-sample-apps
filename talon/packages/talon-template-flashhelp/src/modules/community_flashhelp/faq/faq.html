<template>
    <div class="slds-align_absolute-center slds-text-heading_large" style="height: 5rem;">
        FAQ
    </div>
</template>