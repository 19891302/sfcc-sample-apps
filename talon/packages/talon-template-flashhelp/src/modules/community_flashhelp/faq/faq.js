import { LightningElement } from 'lwc';

export default class Faq extends LightningElement {

}