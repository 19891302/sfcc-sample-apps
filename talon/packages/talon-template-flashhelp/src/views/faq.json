{
    "name": "faq",
    "label": "Frequently Asked Questions",
    "themeLayoutType": "default",
    "component": {
        "name": "community_flashhelp/faq"
    }
}