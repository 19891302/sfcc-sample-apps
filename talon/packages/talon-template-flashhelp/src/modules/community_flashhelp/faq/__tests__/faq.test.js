import { createElement } from 'lwc';
import faq from 'community_flashhelp/faq';

describe('community_flashhelp/faq', () => {
    beforeEach(() => {
        while (document.body.firstChild) {
            document.body.removeChild(document.body.firstChild);
        }
    });

    it('renders correctly', () => {
        const element = createElement('community_flashhelp-faq', { is: faq });
        document.body.appendChild(element);

        return Promise.resolve().then(() => {
            expect(element).toMatchSnapshot();
        });
    });
});
