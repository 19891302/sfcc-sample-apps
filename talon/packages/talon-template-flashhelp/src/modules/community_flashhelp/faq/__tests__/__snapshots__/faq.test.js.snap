// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`community_flashhelp/faq renders correctly 1`] = `
<community_flashhelp-faq>
  #shadow-root(open)
    <div
      class="slds-align_absolute-center slds-text-heading_large"
      style="height: 5rem;"
    >
      FAQ
    </div>
</community_flashhelp-faq>
`;
